storage/google/cloud/storage/notification.py

```diff
@@ -22,6 +22,8 @@
 JSON_API_V1_PAYLOAD_FORMAT = 'JSON_API_V1'
 NONE_PAYLOAD_FORMAT = 'NONE'
 
+_TOPIC_REF = '//pubsub.googleapis.com/projects/{}/topics/{}'
```

Comment:
It is fine to not have a protocol? (I know that is valid now, so not complaining; just ensuring it is an intentional decision.)

Reply:
The format is [specified in the docs for `notifications/insert](https://cloud.google.com/storage/docs/json_api/v1/notifications/insert#topic).